docs/reference/sql/language/syntax/commands/describe-table.asciidoc

```diff
@@ -6,7 +6,9 @@
 .Synopsis
 [source, sql]
 ----
-DESCRIBE [table identifier<1> | [LIKE pattern<2>]]
+DESCRIBE
+    [table identifier | <1>
+    [LIKE pattern]] <2>
```

Comment:
I'd probably vertically align these but it isn't a big deal either way.

Reply:
Thanks for the suggestion @nik9000! I aligned the callouts in c9b9aa6. Also updated the before/after images above.